### ISSUE TITLE
Erased terms doc not exposed

### ISSUE THREAD

**Issue by japgolly** [lampepfl/dotty#10847]
I see that there is https://github.com/lampepfl/dotty/tree/master/docs/docs/reference/metaprogramming/erased-terms.md but it's not accessible from https://dotty.epfl.ch/docs/reference/metaprogramming/toc.html

**Comment by nicolasstucki** [comment]
That is intentional. The implementation of erased is outdated as it does not take into account new language features such as `given`. We would only expose it once this feature is ready for use.

We delayed work on it because it was not part of the features that would appear in 3.0.

**Comment by japgolly** [comment]
Ahhhhh ok. So is it planned for 3.1 then?

**Comment by nicolasstucki** [comment]
It is for `3.x` for some `x`. If time allows it, we aim to have it stabilized `3.1` but still under the flag.

**Comment by abgruszecki** [comment]
@nicolasstucki do we want to keep this issue around then?

**Comment by nicolasstucki** [comment]
We need to find a place to add a link to those docs. Let's keep it for now.

**Comment by michelou** [comment]
@japgolly But it's accessible (as other unreachable sections) in the PDF document [`scala3_reference.pdf`](https://github.com/michelou/dotty-examples/blob/master/docs/scala3_reference.pdf) (internal draft) I generate/update in PR #10767, #10826 and #10860 (more changes are pending) :grin:

**Comment by japgolly** [comment]
np and thanks for the info :slightly_smiling_face:
